bindata/network/ovn-kubernetes/common/008-script-lib.yaml

```diff
@@ -569,6 +569,8 @@ data:
         sysctl -w net.ipv6.conf.all.forwarding=1
       else
         ip_forwarding_flag="--disable-forwarding"
+        sysctl -w net.ipv4.ip_forward=0
+        sysctl -w net.ipv6.conf.all.forwarding=0
```

Comment:
We will need to have the ipv6 enabled even if `IP_FORWARDING_MODE` is not Global.
For more context: https://github.com/ovn-org/ovn-kubernetes/pull/4376
Lets remove that line from here.

Reply:
Okay, Understood. Thanks @kyrtapz 
IPv6 forwarding is also disabled by default just like IPv4 as per https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt.

I do see that you are enabling IPv6 forwarding in your PR(#4376) if either single stack IPv6 or dual stack is configured.

so, I think its okay to have net.ipv6.conf.all.forwarding set to default(0) while we are not using single stack IPv6 or dual stack. Or, should I set net.ipv6.conf.all.forwarding=1 if ipForwarding mode is 'Restricted' instead of removing line 573 completely.

Reply:
@kyrtapz , I have removed the line where I was setting net.ipv6.conf.all.forwarding to '0' when ipForwarding mode is set to 'Restricted'. Please take a look. 
Thanks in advance!